packages/lab/src/calendar/Calendar.tsx

```diff
@@ -37,6 +38,7 @@ export const Calendar = forwardRef<HTMLDivElement, CalendarProps>(
       renderDayContents,
       hideYearDropdown,
       TooltipProps,
+      borderedDropdown,
```

Comment:
Not for this PR...
We can merge this to Labs... but this code will be cherry-picked and refactored.

 This is what I am thinking in the main DatePicker PR

- we make Calendar accept children which allows you to customise the navigation bar (you could put whatever you want there but we can provide CalendarNavigation as a default (in examples).
- we make CalendarNavigation support YearDropdownProps and MonthDropdownProps

As a heads-up, slightly worried about having to add bordered props everywhere, so created this ticket
https://github.com/jpmorganchase/salt-ds/issues/3939